src/core/net/srp_server.cpp

```diff
@@ -1324,6 +1324,50 @@ otError Server::Service::SetFullName(const char *aFullName)
     return error;
 }
 
+otError Server::Service::GetNextTxtEntry(otDnsTxtIterator &aIterator, otDnsTxtEntry &aTxtEntry) const
+{
+    otError error = OT_ERROR_NONE;
+
+    for (uint16_t i = aIterator; i < mTxtLength;)
+    {
+        otDnsTxtEntry entry;
```

Comment:
Was thinking if we can directly populating the `aTxtEntry` fields? Any reason we may not want to?

Reply:
updated.